src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -460,6 +486,18 @@ private static void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> cal
         w.WriteLine("};");
     }
 
+    private bool HasAssemblyDisableRuntimeMarshallingAttribute(Assembly assembly)
+    {
+        if (!_assemblyDisableRuntimeMarshallingAttributeCache.TryGetValue(assembly, out var value))
+        {
+            _assemblyDisableRuntimeMarshallingAttributeCache[assembly] = value = assembly
+                .GetCustomAttributesData()
+                .Any(d => d.AttributeType.Name == "DisableRuntimeMarshallingAttribute");
```

Comment:
can use `nameof(DisableRuntimeMarshallingAttribute)` with `using System.Runtime.CompilerServices;`

Reply:
The assembly is also compiled for net47 and the attribute is not available there.